srcs/Clientfile.cpp

```diff
@@ -37,4 +45,29 @@ void Clientfile::on_write(){
 
 const Client * Clientfile::getClient() const{
     return this->_client;
-}
\ No newline at end of file
+}
+
+void Clientfile::appendToBuffer(const std::string& data)
+{
+    _buffer += data;
+}
+
+const std::vector<std::string> Clientfile::flushBuffer()
+{
+    std::vector<std::string> flushed;
+
+    std::size_t pos_start = 0;
+    std::size_t pos_end;
+    while (true)
+    {
+        pos_end = _buffer.find('\n');
```

Comment:
```suggestion
        pos_end = _buffer.find("\r\n");
```
CR LF가 기준이므로 일단은 캐리지리턴을 포함하여 구분하도록 변경해둡시다.